README.md

```diff
@@ -91,20 +91,27 @@ Zonemaster comes with localization for these locales:
 
 ## Repositories
```

Comment:
I think the "Repositories" should be renamed "Components", and the language in the section contents be updated to describe things in terms of components.

Reply:
Updated.